CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+## Contributing
+
+Welcome to make your contribution to Gluten project!
+
+The contribution is not limited to contributing code. It also includes reviewing code, improving documentation, proposing ideas, etc.
+
+### GitHub Pull Request
+
+If necessary, please file a GitHub issue for your PR and refer to that issue in your PR title.
+
+By convention, `CH` & `VL` represent ClickHouse & Velox backend respectively. Please add `[CH]` or `[VL]` in your PR title
+accordingly if your proposed code change is just for one specific backend. Please do NOT do this if it is for common module,
+independent of any backend.
+
+Examples:
+* PR just for CH backend:
+
+  `[GLUTEN-<issue ID>][CH] xxxx`
+
+* PR just for VL backend:
+
+  `[GLUTEN-<issue ID>][VL] xxxx`
+
+* PR for common module:
+
+  `[GLUTEN-<issue ID>] xxxx`
```

Comment:
Just to clarify. Does this mean there is no tab for ```[CORE]``` moving forward? Also, will Github issue a must?

Reply:
I think it would be better to add `[CORE]` to mention if the code change affect both Velox and CH.

Reply:
> Also, will Github issue a must

I think a issue is required for several cases:
1. we are going to send a batch of prs for one topic, means it's a umbrella
2. a bug report or feature improvement that the reporter won't send pr directly or eagerly

Then for others, it is optional. But the description is required, developers should put something into pr description to make pr easy to follow.

Reply:
Yes, github issue is not mandatory. `if necessary` is used in the above statement.